boot/zephyr/Kconfig

```diff
@@ -43,6 +44,29 @@ config BOOT_USE_TINYCRYPT
 	help
 	  Use TinyCrypt for crypto primitives.
 
+config BOOT_USE_CC310
+	bool
+	# Hidden option
+	default n
+	# When building for ECDSA, we use our own copy of mbedTLS, so the
+	# Zephyr one must not be enabled or the MBEDTLS_CONFIG_FILE macros
+	# will collide.
+	depends on ! MBEDTLS
+	help
+	  Use cc310 for crypto primitives.
+
+config BOOT_USE_NRF_CC310_BL
+	bool
+	default n
+
+config NRFXLIB_CRYPTO
+	bool
+	default n
+
+config NRF_CC310_BL
```

Comment:
I think this is redundant I'll fix it upstream as it's defined sort of twice in L93-94

Reply:
ok

Reply:
can you do it quickly here as [nrf fromlist] commit and send path to upsterem as well?

Reply:
This is still in two places NCS/nrfxlib/crypto/Kconfig and NCS/mcuboot/boot/zephyr/Kconfig

Reply:
@sigvartmh  ^^

Reply:
It should be in  NCS/nrfxlib/crypto/Kconfig but not in  NCS/mcuboot/boot/zephyr/Kconfig I thought I removed it.

Reply:
Do you want me to fix it maybe? then make a PR towards your branch with changes to the remove duplicate commit?

Reply:
@sigvartmh  fix on my branch, I'll give you permissions.

Reply:
okei :+1:

Reply:
Fixed @nvlsianpu